design/ocwg-design-decisions.md

```diff
@@ -126,33 +76,34 @@ Discussed in [2](https://github.com/orgs/ocwg/discussions/2).
 | originalText | `"Hello World"` | *collaboration* |  |
 | lineHeight | `1.25` | *text style* |  |
 
-## Zwibbler
 
-See [References](#_references) for a sheet on properties and node types.
+### Zwibbler
+See [References](#references) for a sheet on properties and node types.
+
+
 
-# Design Decisions
 
-## Nodes vs. Resources
 
+## Design Decisions
+
+### Nodes vs. Resources
 We separate the visual elements (nodes) of a canvas from the displayed
 content (resources). This allows displaying the same resource in
 multiple places, which is great for e.g. block arrow symbols. This in
-turn saves space and makes the format more [Human Friendly](#human).
+turn saves space and makes the format more human-friendly (see goals).
 
 SVG also provides
 [defs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/defs)
 and use tags.
 
-## Resources are Web-like
-
+### Resources are Web-like
 We take the way the web uses resources (URI + content negotiation →
 representation + content type) as inspiration to model resources.
 
-## Resource Structure
-
+### Resource Structure
 We need to be able to refer to resources using a locally stable ID.
-Taking [Resources are Web-like](#resource), [Offline](#offline) and
-[Graceful Degration](#resource-fallback) into account, we get this
+Taking the requirements "Resources are Web-like", "Offline" and
+"Graceful Degration" into account, we get this
 minimal structure for resources:
```

Comment:
I didn't understand the "minimal structure" that diagram until I went and read the "Resources" section of the spec. This "design decision" probably needs a _bit_ more pre-amble.

Reply:
Ok, thanks, I improved that section